testutil/src/test/java/org/spine3/test/TestsShould.java

```diff
@@ -97,6 +97,32 @@ public void return_empty_StreamObserver() {
         emptyObserver.onCompleted();
     }
 
+    @Test(expected = AssertionError.class)
+    public void have_own_assertion_for_boolean() {
+        // This should pass.
+        assertEquals(true, true);
+
+        // This should pass too.
+        assertEquals(false, false);
+
+        // This should fail.
+        Tests.assertEquals(true, false);
+    }
+
+    @SuppressWarnings("ConstantConditions") // The call with `false` should always fail.
+    @Test(expected = AssertionError.class)
+    public void have_own_assertTrue() {
+        // This should pass.
+        Tests.assertTrue(true);
+
+        // This should fail.
+        Tests.assertTrue(false);
```

Comment:
This test case in error-prone. Both the first and the second statement may cause an `AssertionError`. 
Try using `try-catch` instead. This may help us to determine that the `AssetionError` is thrown from the second statement.